cooltools/cli/compute_expected.py

```diff
@@ -104,6 +143,8 @@ def compute_expected(
         weight2 = weight_name + "2"
         transforms = {"balanced": lambda p: p["count"] * p[weight1] * p[weight2]}
     else:
+        # no masking bad bins of any kind, when balancing is not applied
+        weight_name = None
```

Comment:
when `--no-balance` - there will be no masking until `blacklist` is ready
how else can we do it ?